### PR TITLE
Update JS documentation for crypto library

### PR DESCRIPTION
**Pull request by arkmoon**
## Summary

Updates the JS docs for using the latest version of the Crypto library as of 

`@arkecosystem/crypto@2.5.26`

### CODE REVIEW

**Comment by ghost** [comment]
**Thanks for submitting this pull request!** A maintainer will review this in the next few days and explicitly select labels so you know what's going on.

_If no reviewer appears after a week, a reminder will be sent out._

File: docs/sdk/cryptography/usage.md [hunk 1]
```diff
@@ -1029,7 +1031,7 @@ A serialized transaction may be deserialized for inspection purposes. The public
 ::: tab javascript
 
 ```js
-const deserialized = Transaction.deserialize(serialized);
+const deserialized = Transactions.deserializer.deserialize(serialized);
```
Comment (dated) [inline docs/sdk/cryptography/usage.md]:
```suggestion
const deserialized = Transactions.Deserializer.deserialize(serialized);
```

Comment (arkmoon) [reply docs/sdk/cryptography/usage.md]: @dated :
I actually wanted to bring this up. `Transactions.Deserializer` is undefined. It looks like it's exporting an instance of `Transactions.deserializer` as a const. It's not exporting the class like `Serializer` does.

**Comment by dated** [comment]
Oh, my bad, sorry about the confusion. It should probably stay lowercased then.

File: docs/sdk/cryptography/usage.md (hunk 1 above)
Comment (arkmoon) [reply docs/sdk/cryptography/usage.md]: Fixed.

**Comment by dated** [comment]
FYI https://github.com/ArkEcosystem/core/pull/3234 :pray:

File: docs/sdk/cryptography/usage.md [hunk 2]
```diff
@@ -600,13 +600,15 @@ Depending on the library, you must import and initialize it in a specific way.
...
 ```js
-const { crypto } = require("@arkecosystem/crypto");
+const { Crypto } = require("@arkecosystem/crypto");
 ```
 
 Throughout this document, the keys object used is:
 
 ```js
-const keys = crypto.getKeys("this is a top-secret passphrase");
+const BIP32Wallet = Crypto.HDWallet.fromMnemonic("this is a top-secret passphrase");
```
Comment (faustbrian) [inline docs/sdk/cryptography/usage.md]: This can be removed because of below comment.

File: docs/sdk/cryptography/usage.md [hunk 3]
```diff
@@ -600,13 +600,15 @@ Depending on the library, you must import and initialize it in a specific way.
...
+
+const keys = Crypto.HDWallet.getKeys(BIP32Wallet);
```
Comment (faustbrian) [inline docs/sdk/cryptography/usage.md]:
```suggestion
const keys = Identities.Keys.fromPassphrase("this is a top secret passphrase");
```

**Comment by ItsANameToo** [comment]
@arkmoon any ETA to look at the changes Brian requested and to fix the conflict?

File: docs/sdk/cryptography/usage.md (hunk 2 above)
Comment (arkmoon) [reply docs/sdk/cryptography/usage.md]: I replaced these two lines with:
`const keys = Identities.Keys.fromPassphrase("this is a top secret passphrase");` as requested

**Review by arkmoon — COMMENTED** [review]
Made suggested changes to my branch.

**Comment by arkmoon** [comment]
I pushed the commits to resolve the conversation, but not seeing them showing up here in the files changed.

**Comment by ghost** [comment]
Your pull request has been closed, thank you for trying to solve an issue. If you think it was closed prematurely please provide additional information.

**Pull request closed without merging**